azure-pipelines.yml

```diff
@@ -84,6 +84,9 @@ jobs:
         # acquire access token from Azure CLI and export it to AZURE_MANAGEMENT_ACCESS_TOKEN
         $accessToken = (az account get-access-token --query "accessToken" | % { $_.Trim('"') })
         echo "##vso[task.setvariable variable=azure_management_access_token]$accessToken"
+  - pwsh: |
```

Comment:
Should this task only be run for release builds?

Reply:
My preference is to run during PR builds to prevent language owners from updating just their worker instead of all host/workers at once. I think this keeps our branches in more of a "shippable" state

Reply:
Good point. Let's keep it as is.